doc/source/contribution_and_api/building_example.rst

```diff
@@ -0,0 +1,62 @@
+.. _ref_example_building_api:
+
+Building an example
+===================
+
+
+To run the documentation, you need to have the correct versions of each tools. To do so, execute the following instruction.
+
+.. code:: bash
+
+    pip install -r requirements/requirements_docs.txt
+
+
+The Sphinx configuration is in the file ``conf.py`` in ``doc/source``.
+
+To run the sphinx tool:
+
+.. code:: bash
+
+    doc\make.bat html
+
+
+
+There are three types of examples: dynamic, static, and semi-static.
+
+
+Dynamic examples
+----------------
+
+They are based on Python files and must be run under 3 minutes.
+
+They are in the ``examples`` directory inside this repository.
+
+Example: https://github.com/pyansys/pymapdl/blob/main/examples/00-mapdl-examples/2d_plate_with_a_hole.py
+
+It is published on the following link: 
+`MAPDL 2D Plane Stress Concentration Analysis <https://mapdl.docs.pyansys.com/examples/gallery_examples/00-mapdl-examples/2d_plate_with_a_hole.html#sphx-glr-examples-gallery-examples-00-mapdl-examples-2d-plate-with-a-hole-py>`_
+
+They will be executed and the mention **Total running time of the script** will appear at the end of the document.
+
+
+Static examples
+---------------
+
+They are based on RST files and will not be executed. 
+
+They are in the ``doc\source`` directory.
+
+Example: https://raw.githubusercontent.com/pyansys/pymapdl/main/doc/source/examples/extended_examples/Krylov/krylov_example.rst
+
+It is published on the following link: `Harmonic analysis using the frequency-sweep Krylov method <https://mapdl.docs.pyansys.com/examples/extended_examples/Krylov/krylov_example.html>`_
+
+
+Semi-dynamic examples
+---------------------
+
+They are RST files which execute Python code.
+
```

Comment:
Can you add the info regarding which RST directive is used to run this python code?